gtsam/hybrid/tests/testDCGaussianMixtureFactor.cpp

```diff
@@ -99,7 +99,7 @@ TEST(DCGaussianMixtureFactor, GaussianMixture) {
 
   auto gaussians = boost::make_shared<GaussianConditional>();
   GaussianMixture::Conditionals conditionals(gaussians);
-  GaussianMixture gm(keys, dKeys, conditionals);
+  GaussianMixture gm(0, keys, dKeys, conditionals);
```

Comment:
Does not make sense to me

Reply:
GaussianConditional here is empty (invalid)